libs/blocks/merch-card/merch-card.js

```diff
@@ -49,10 +52,9 @@ const appendSlot = (slotEls, slotName, merchCard) => {
   merchCard.append(newEl);
 };
 
-const parseContent = (el, merchCard) => {
-  const innerElements = [
-    ...el.querySelectorAll('h2, h3, h4, h5, p, ul, em'),
-  ];
+const parseContent = async (el, merchCard) => {
+  const allElements = el.querySelectorAll('h2, h3, h4, h5, p, ul, em');
+  const innerElements = Array.from(allElements).filter((element) => !element.closest('.mnemonic-list'));
```

Comment:
this is getting complicated.
Let's first process mnemonic, then you would not need to change this part.

Reply:
Updated